package.json

```diff
@@ -139,6 +139,16 @@
     "@cheap-glitch/mi-cron": "1.0.1",
     "@iarna/toml": "2.2.5",
     "@jamiemagee/osv-offline": "1.2.6",
+    "@opentelemetry/api": "1.1.0",
+    "@opentelemetry/context-async-hooks": "1.2.0",
+    "@opentelemetry/exporter-trace-otlp-http": "0.28.0",
+    "@opentelemetry/instrumentation": "0.28.0",
+    "@opentelemetry/instrumentation-bunyan": "0.28.0",
+    "@opentelemetry/instrumentation-http": "0.28.0",
+    "@opentelemetry/resources": "1.2.0",
+    "@opentelemetry/sdk-trace-base": "1.2.0",
+    "@opentelemetry/sdk-trace-node": "1.2.0",
+    "@opentelemetry/semantic-conventions": "1.2.0",
```

Comment:
It's curious to see there's sync'd 0.x dependencies here, also some synced 1.2.0 dependencies, and one 1.1.0 dependency. What's the opentelemetry approach to releasing, 1.0.0 releases, etc?

Reply:
`@opentelemetry/api` has its own [repository](https://github.com/open-telemetry/opentelemetry-js-api) and is released separately. 

The dependencies with version `1.2.0` come all from the core [opentelemetry-js repository](https://github.com/open-telemetry/opentelemetry-js) 

That all pre-1.0.0 dependencies have version `0.28.0` is by chance. They originate from the core repository and the [contrib one](https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/plugins/node) which contains community contributions